packages/dev/src/quickstarts-data/yaml/openshift-tabbed-copy.yaml

```diff
@@ -35,6 +35,9 @@ spec:
           ```{{copy}}
 
         2. In the main navigation menu, click **Add.**
+      
+          [This can be an accordion input, followed by another]{{accordion "testing with more text"}}
```

Comment:
Should this work with multiline text as well? Like when you want to create a list within the accordion